src/components/BusinessPosts/Post.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+
+interface Media {
+    mediaId: bigint;
+    postId: bigint;
+    mediaUrl: string;
+    mediaType: string;
+    createdAt: string;
+}
+
+interface PostProps {
+    post: {
+        postId: bigint;
+        userId: bigint;
+        title: string;
+        content: string;
+        createdAt: string;
+        updatedAt?: string;
+    };
+    media: Media[];
+    onEdit: (postId: bigint, title: string, content: string) => void;
+    onDelete: (postId: bigint) => void;
+    isEditing: boolean;
+    editTitle: string;
+    editContent: string;
+    setEditTitle: (title: string) => void;
+    setEditContent: (content: string) => void;
+    handleUpdate: (event: React.FormEvent) => void;
+}
```

Comment:
We want to store these type definitions in `/types` or in a location where they are globally accessible